frontend/src/metabase/visualizations/components/ClickActions/ClickActionsPopover.tsx

```diff
@@ -49,13 +51,14 @@ export class ClickActionsPopover extends Component<
   };
 
   handleClickAction = (action: RegularClickAction) => {
-    const { dispatch, onChangeCardAndRun } = this.props;
+    const { dispatch, onChangeCardAndRun, onUpdateQuestion } = this.props;
     if (isPopoverClickAction(action)) {
       this.setState({ popoverAction: action });
     } else {
       const didPerform = performAction(action, {
         dispatch,
         onChangeCardAndRun,
+        onUpdateQuestion,
```

Comment:
Allows updating the question without running. Looks like there is a lot going on in `onChangeCardAndRun`, so I didn't want to add a `run` flag there - best to keep them separate.